engine/CMakeLists.txt

```diff
@@ -1264,5 +1264,5 @@ IF (USE_GTEST)
     )
 
     INCLUDE(GoogleTest)
-    gtest_discover_tests(${TEST_NAME})
+    gtest_discover_tests(${TEST_NAME} PROPERTIES DISCOVERY_TIMEOUT 30)
```

Comment:
Builds have started failing more and more often on my local machine with a timeout error discovering tests. Turns out that the default timeout is only 5 seconds. This increases it to 30. No more timeouts, at least not that I've seen.